docs/translations/api-docs/data-grid/grid-toolbar-button/grid-toolbar-button.json

```diff
@@ -0,0 +1,20 @@
+{
+  "componentDescription": "",
+  "propDescriptions": {
```

Comment:
Is there a way I can get the props table on this API page to pull in the [ButtonBase props](https://mui.com/material-ui/api/button-base/#props), since it extends that interface?